packages/sse/src/decorators.ts

```diff
@@ -0,0 +1,12 @@
+import type { TerminalEventDecorator } from "../generated-defs/TypeSpec.SSE.js";
+import { SSEStateKeys } from "./lib.js";
+
+/** @internal */
+export const namespace = "TypeSpec.SSE";
```

Comment:
also shouldn'ty need namespace anymore if you use $decorators

Reply:
Nice - couldn't actually tell how this was being used before so happy to remove!